index.html

```diff
@@ -1,232 +1,320 @@
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <header>
+      <nav class="top-sec">
+        <a href="#"><h1 class="nav-branding">Oscar Deus</h1></a>
+        <ul class="nav-menu">
+          <li class="nav-item nav-link"><a href="#">Portfolio</a></li>
+          <li class="nav-item nav-link"><a href="#">About</a></li>
+          <li class="nav-item nav-link"><a href="#">Contact</a></li>
+        </ul>
+        <div class="hambuger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+      </nav>
+    </header>
+
+    <section class="homepage">
+      <div class="headtitle-name headtitle">
+        <h1 class="primary-text">I'm Oscar<br />Glad to see you!</h1>
+        <p class="phar">
+          I'm a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don't hestiate to contact
+          me.
+        </p>
+        <h3 class="letsconnect">LET'S CONNECT</h3>
+
+        <ul class="socialmedia-list">
+          <li><img src="images/Vector.png" alt="vector1" /></li>
+           <li><img src="images/Linkedinicon.png" alt="vectorlink" /></li>
+          <li class="socials"><img src="images/med.png" alt="med" /></li>
+          <li class="socials"><img src="images/cat.png" alt="cat" /></li>
+          <li class="socials"><img src="images/angelist.png" alt="ange" /></li>
+        </ul>
+      </div>
+    </section>
+
+    <section class="container">
+      <section class="works-temp1 works1">
+        <img src="images/SnapshootPortfolio(1).png" class="pic-temp1" alt="" />
+        <div class="works-info1">
+          <h1 class="tonic">Tonic</h1>
+          <ul class="work-info">
+            <li class="canopy">CANOPY</li>
+            <li class="back-end-dev">
+              <img src="images/Counter(1).png" alt="" />
+              <span>Back End Dev</span>
+            </li>
+            <li class="back-end-dev">
+              <img src="images/Counter(1).png" alt="" />
+              <span>2015</span>
+            </li>
+          </ul>
+          <p class="proj-info">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="prog-lang">
+            <li class="lang-styles">HTML</li>
+            <li class="lang-styles">Css</li>
+            <li class="lang-styles">JavaScript</li>
+          </ul>
+          <button type="button" class="btn">See Projects</button>
+        </div>
+      </section>
```

Comment:
- [ ] `Optional`
- [ ] Please replace the image with the correct image on the figma
- [ ] Please add some margin or padding in accordance to the figma templet see screenshots below

`your image`
![Screenshot from 2022-07-16 15-35-03](https://user-images.githubusercontent.com/66230306/179357425-fb418c80-d323-4021-a8ec-3ec4ade4844d.png)

`figma template`
![Screenshot from 2022-07-16 15-44-39](https://user-images.githubusercontent.com/66230306/179357520-be189f45-3029-4a7b-8f7c-c7409398afe3.png)